test/benchmark/config/gatewayclass.yaml

```diff
@@ -0,0 +1,84 @@
+kind: GatewayClass
+apiVersion: gateway.networking.k8s.io/v1
+metadata:
+  name: envoy-gateway
+spec:
+  controllerName: gateway.envoyproxy.io/gatewayclass-controller
+  parametersRef:
+    group: gateway.envoyproxy.io
+    kind: EnvoyProxy
+    name: proxy-config
+    namespace: envoy-gateway-system
+---
+apiVersion: gateway.envoyproxy.io/v1alpha1
+kind: EnvoyProxy
+metadata:
+  name: proxy-config
+  namespace: envoy-gateway-system
+spec:
+  provider:
+    type: Kubernetes
+    kubernetes:
+      envoyDeployment:
+        container:
+          volumeMounts:
```

Comment:
is the ext proc config required ?

Reply:
no. this content is directly copied from https://github.com/envoyproxy/gateway/blob/main/test/config/gatewayclass.yaml w/o any modifications.

I will check this file again, add some resource limits and remove useless configs.

Reply:
removed